runtime/src/lib.rs

```diff
@@ -1400,7 +1401,7 @@ impl EnsureOrigin<Origin> for EnsureRootOrPolkadexTreasury {
 
 impl orml_vesting::Config for Runtime {
 	type Event = Event;
-	type Currency = pallet_balances::Module<Runtime>;
+	type Currency = pallet_balances::Pallet<Runtime>;
```

Comment:
This is for an deprecation warning @frankli-dev  do you think it is related to #335 ?

Reply:
I'm not sure...